### ISSUE TITLE
Tradeup contract automation?

### ISSUE THREAD

**Issue by Andrew3019** [DoctorMcKay/node-globaloffensive#57]
Possible to automate the trade-up contract?

**Comment by unrandomizer** [comment]
Yep, it's possible.  I would be glad if you drop me a line on steam

**Comment by Tankonyako** [comment]
> Possible to automate the trade-up contract?

i can send code for automation crafting in csgo, but not free.

**Comment by Andrew3019** [comment]
> > Possible to automate the trade-up contract?
> 
> i can send code for automation crafting in csgo, but not free.

How can I contact you?

**Comment by Tankonyako** [comment]
> > > Possible to automate the trade-up contract?
> > 
> > 
> > i can send code for automation crafting in csgo, but not free.
> 
> How can I contact you?

ftsu#3507

**Comment by DarkGL** [comment]
https://github.com/DoctorMcKay/node-globaloffensive/issues/15

**Comment by DarkGL** [comment]
https://github.com/DoctorMcKay/node-globaloffensive?tab=readme-ov-file#craftitems-recipe
